yb-voyager/cmd/common.go

```diff
@@ -480,21 +447,16 @@ func nameContainsCapitalLetter(name string) bool {
 }
 
 func getCutoverStatus() string {
-	cutoverFpath := filepath.Join(exportDir, "metainfo", "triggers", "cutover")
-	cutoverSrcFpath := filepath.Join(exportDir, "metainfo", "triggers", "cutover.source")
-	cutoverTgtFpath := filepath.Join(exportDir, "metainfo", "triggers", "cutover.target")
-	fallforwardSynchronizeStartedFpath := filepath.Join(exportDir, "metainfo", "triggers", "fallforward.synchronize.started")
-
-	a := utils.FileOrFolderExists(cutoverFpath)
-	b := utils.FileOrFolderExists(cutoverSrcFpath)
-	c := utils.FileOrFolderExists(cutoverTgtFpath)
-	d := utils.FileOrFolderExists(fallforwardSynchronizeStartedFpath)
-
-	migrationStatusRecord, err := metaDB.GetMigrationStatusRecord()
+	msr, err := metaDB.GetMigrationStatusRecord()
 	if err != nil {
-		utils.ErrExit("could not fetch MigrationstatusRecord: %w", err)
+		utils.ErrExit("get migration status record: %v", err)
 	}
-	ffDBExists := migrationStatusRecord.FallForwarDBExists
+
+	a := msr.IsTriggerExists("cutover")
+	b := msr.IsTriggerExists("cutover.source")
+	c := msr.IsTriggerExists("cutover.target")
+	d := msr.IsTriggerExists("fallforward.synchronize.started")
```

Comment:
As suggested, let's replace these with descriptive flag fields in the `MSR` itself.